src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/Datagrid.js

```diff
@@ -82,6 +85,37 @@ class Datagrid extends React.Component<ViewProps> {
         router.bind('limit', this.datagridStore.limit, 10);
     }
 
+    @action componentDidMount() {
+        const {router} = this.props;
+        const {
+            route: {
+                options: {
+                    locales,
```

Comment:
this id different to Form.js not sure if this is the correct way

Reply:
I think the way you have done it is fine. The thing is that in the `Form` component the locales can either be passed by route options or from the parent view (usually the `ResourceTabs` or `PageTabs` component), and one of them take precedence. However, in the case of the `Form` there is a computed property called `locale`, which should be used for this. Since you are already at it, can you maybe fix that as well?

Reply:
Fixed